bootstrap/helpers.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+use App\Models\User;
+
+/**
+ * Get the currently authenticated user model.
+ */
+function user(): User
+{
+    $currentUser = auth()->user();
+
+    if ($currentUser === null) {
+        throw new Exception('The current user is not authenticated.');
+    }
```

Comment:
If you want, you could shorten this
```suggestion
    throw_if($currentUser === null, Exception::class, 'The current user is not authenticated.');
```